Source/ProjectFirma.Web/Views/Vendor/Detail.cshtml

```diff
@@ -0,0 +1,293 @@
+﻿@*-----------------------------------------------------------------------
+    <copyright file="Detail.cshtml" company="Tahoe Regional Planning Agency and Sitka Technology Group">
+    Copyright (c) Tahoe Regional Planning Agency and Sitka Technology Group. All rights reserved.
+    <author>Sitka Technology Group</author>
+    </copyright>
+
+    <license>
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU Affero General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU Affero General Public License <http://www.gnu.org/licenses/> for more details.
+
+    Source code is available upon request via <support@sitkatech.com>.
+    </license>
+    -----------------------------------------------------------------------*@
+@using ProjectFirma.Web.Models
+@using ProjectFirma.Web.Views
+@using ProjectFirma.Web.Views.Shared
+@using LtInfo.Common
+@using LtInfo.Common.BootstrapWrappers
+@using LtInfo.Common.DhtmlWrappers
+@using LtInfo.Common.HtmlHelperExtensions
+@using LtInfo.Common.ModalDialog
+@using ProjectFirma.Web.Common
+@using ProjectFirma.Web.Controllers
+@using ProjectFirma.Web.Security
+@using ProjectFirma.Web.Views.Vendor
+@inherits ProjectFirma.Web.Views.Vendor.Detail
+
+@functions
+{
+
+}
+
+@section JavascriptAndStylesContent
+{
+    @{ DhtmlxGridIncludes.RenderPartialView(Html); }
+    <script type="text/javascript" src="/Content/Bootstrap-select/bootstrap-select.min.js"></script>
+    <link href="~/Content/bootstrap-tags/bootstrap-tags.min.css" rel="stylesheet" type="text/css" />
+    <link rel="stylesheet" href="/Content/Bootstrap-select/bootstrap-select.min.css" type="text/css" />
+    <script src="@Url.Content("~/Content/bootstrap-toc/bootstrap-toc.min.js")" type="text/javascript"></script>
+}
+
+<style type="text/css">
+    /*Hack to get scrollspy to select sections with no content*/
+    footer {
+        height: 600px;
+    }
+
+    #editLocationDropdownButton:focus {
+        text-decoration: none;
+    }
+
+    .basicsLayout .row {
+        margin-left: 0;
+        margin-right: 0;
+    }
+
+        .basicsLayout .row div {
+            padding-left: 0;
+            padding-right: 5px;
+        }
+
+    .basicsLayout hr {
+        margin-top: 5px;
+        margin-bottom: 5px;
+    }
+</style>
+
+@section RightOfPageTitle
+{
+
+}
+
+<div data-spy="scroll" data-target="#toc">
+    <div id="contentForTOC" class="container-fluid">

```

Comment:
Lets rip out all the TOC stuff. This is a pattern we only use on the Project page, because that one is very long. This page shouldn't grow any larger and we should make it consistent with the other detail pages